### PR TITLE
fix: social buttons style

### PR DESCRIPTION
**Pull request by alisher-aituarov**
## Summary of change

Fixes buttons hover cursor not being changed to pointer issue

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Bonus points for screenshots and videos!)

## Documentation changes

(If relevant, please create a PR in our [docs repo](https://github.com/supertokens/docs), or create a checklist here highlighting the necessary changes)

## Checklist for important updates

-   [x] Changelog has been updated
-   [ ] `frontendDriverInterfaceSupported.json` file has been updated (if needed)
-   [ ] Changes to the version if needed
    -   In `package.json`
    -   In `package-lock.json`
    -   In `lib/ts/version.ts`
-   [x] Had run `npm run build-pretty`
-   [x] Had installed and ran the pre-commit hook
-   [x] Issue this PR against the latest non released version branch.
    -   To know which one it is, run find the latest released tag (`git tag`) in the format `vX.Y.Z`, and then find the latest branch (`git branch --all`) whose `X.Y` is greater than the latest released tag.
    -   If no such branch exists, then create one from the latest released branch.
-   [ ] If added a new recipe interface, then make sure that the implementation of it uses NON arrow functions only (like `someFunc: function () {..}`).
-   [ ] If I added a new recipe, I also added the recipe entry point into the `size-limit` section of `package.json` with the size limit set to the current size rounded up.
-   [ ] If I added a new recipe, I also added the recipe entry point into the `rollup.config.mjs`

### CODE REVIEW

**Review by porcellus — DISMISSED** [review]
LGTM

**Comment by rishabhpoddar** [comment]
@alisher-aituarov please resolve conflict

File: lib/ts/recipe/thirdparty/components/themes/styles.css [hunk 1]
```diff
@@ -9,6 +9,10 @@
     padding-bottom: 9px;
 }
 
+[data-supertokens~="providerContainer"] button {
+    cursor: pointer;
```
Comment (rishabhpoddar) [inline lib/ts/recipe/thirdparty/components/themes/styles.css:13]: can you confirm that if the user provides their own custom component (with a custom provider, or with the google provider), the pointer style is still applied?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: lib/ts/styles/styles.css [hunk 2]
```diff
@@ -234,6 +234,7 @@
     background-position: center;
     transition: all 0.4s;
```
Comment (rishabhpoddar) [inline lib/ts/styles/styles.css:235]: is it possible to add unit test for this?

**Review by rishabhpoddar — CHANGES_REQUESTED** [review]

File: lib/ts/recipe/thirdparty/components/themes/styles.css (hunk 1 above)
Comment (alisher-aituarov) [reply lib/ts/recipe/thirdparty/components/themes/styles.css:13]: it will be applied if user uses button element

Comment (rishabhpoddar) [reply lib/ts/recipe/thirdparty/components/themes/styles.css:13]: The user is not supposed to use the button element in their custom component cause the click handler is applied by us. Please make sure you understand how things work

**Comment by alisher-aituarov** [comment]
Screen Shot 2023-02-02 at 17 00 23
Screen Shot 2023-02-02 at 17 02 17
Screen Shot 2023-02-02 at 17 02 53

File: lib/ts/styles/styles.css (hunk 2 above)
Comment (alisher-aituarov) [reply lib/ts/styles/styles.css:235]: it is possible, discussed this with @porcellus and this seem to be a lot of work to just to check that line of css does work

Comment (porcellus) [reply lib/ts/styles/styles.css:235]: The other big issue with the test is that we'd be testing that CSS rules work, but as far as I can tell, there is no way to test something like "what's the cursor like if I hover over this part of the page/button".

**Review by porcellus — APPROVED** [review]

**Review by rishabhpoddar — APPROVED** [review]

**Pull request merged**